### ISSUE TITLE
Callback triggered twice

### ISSUE THREAD

**Issue by brianshen1990** [adaltas/node-hbase#65]
Hadoop Version: 3.2.0
HBase Version: 2.1.3

```
const hbase = require('hbase')
// Instantiate a new client
client = hbase({ host: '10.21.137.43', port: 60080 });
const myRow = new hbase.Row(client, 'webtable', 'com.cnn.www')
myRow.get('contents', (err, value) => {
  console.log('==contents======');
  console.log(err);
  console.log(value);
});
```

Results:
```
==contents======
null
[ { column: 'contents:html',
    timestamp: 1552356956216,
    '$': '<html>...02' },
  { column: 'contents:key01',
    timestamp: 1552374290762,
    '$': 'my value 1' },
  { column: 'contents:key02',
    timestamp: 1552374290762,
    '$': 'my value 2' } ]
==contents======
Error: Connection closed
    at IncomingMessage.<anonymous> (/Users/BrianShen/Code/Brian/tset/Node/hbase/node_modules/hbase/lib/connection.js:130:13)
    at IncomingMessage.emit (events.js:182:13)
    at Socket.socketCloseListener (_http_client.js:355:11)
    at Socket.emit (events.js:187:15)
    at TCP._handle.close (net.js:606:12)
null
```

**Comment by brianshen1990** [comment]
![Screen Shot 2019-03-12 at 3 28 01 PM](https://user-images.githubusercontent.com/12724940/54182066-67b5b900-44db-11e9-8346-d0308f32c699.png)

I notices both close and end will trigger callback, so it is designed in this way?

**Comment by wdavidw** [comment]
Instead of a screenshot, please show provide a github url because the code you are mentionning does not exist in the [current HEAD](https://github.com/adaltas/node-hbase/blob/master/lib/connection.js).

**Comment by brianshen1990** [comment]
Line 121, 136

**Comment by wdavidw** [comment]
There is no 'close' event in those lines.

**Comment by brianshen1990** [comment]
![Screen Shot 2019-03-13 at 9 46 42 AM](https://user-images.githubusercontent.com/12724940/54247694-dbf06b00-4574-11e9-8a2b-f9b15004182a.png)

**Comment by brianshen1990** [comment]
Sorry for that, My hbase version is 0.4.2. And npm downloaded it by default. in both 0.4.2 and 0.5.0, there is close event, however, in master branch, i notice that this problem has been fixed. 

Thanks

**Comment by wdavidw** [comment]
Seems like master and NPM have diverged. Also, I just realized that version 0.5 publication was a mistake, it was published before 0.4.1 and actually was not meant to be published. To fix it, I'll publish both 0.4.3 and 0.5.1.

Also, @IDerr your latest commit on 20 February is not correct for now. I sent a pull [request](https://github.com/nerdammer/dockers/pull/7) to the author of the nerdammer/hbase image on which sixeyed/hbase-stargate rely on. It was accepted but the image on Docker Hub was not updated, either we revert and use our own fork or we wait for the update of nerdammer/hbase on Docker Hub. I just sent a message with this request.

**Comment by wdavidw** [comment]
Finally, I've published 0.6 to make think clear. Using `sixeyed/hbase-stargate` and indirectly `nerdammer/hbase` work on my host, macos, but from memory, it shouldn't work on a recent Linux kernel unless I'm missing something.

**Comment by IDerr** [comment]
Thanks @wdavidw for your work, I think it's better to wait a bit for the updated image on Docker Hub.

Sorry for the extra work.
